crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -97,6 +99,63 @@ pub fn conv2d_config(curr: &Node) -> Conv2dConfig {
     .with_padding(padding)
 }
 
+/// Create a Conv3dConfig from the attributes of the node
+pub fn conv3d_config(curr: &Node) -> Conv3dConfig {
+    let mut kernel_shape = Vec::new(); // TODO default inferred from weight tensor per spec
+    let mut strides = vec![1, 1, 1];
+    let mut pads = vec![0, 0, 0, 0, 0, 0];
+    let mut dilations = vec![1, 1, 1];
+    let mut group: i64 = 1;
+
+    // extract the channels from the weight tensor's shape [out_channels, in_channels, ...]
+    let weight = if let ArgType::Tensor(ref weight) = curr.inputs[1].ty {
+        weight
+    } else {
+        panic!("Conv3d: weight tensor must be present");
+    };
+    // check if the bias is present
+    let bias = curr.inputs.len() == 3;
+
+    // the channels are inverted in the weight tensor
+    let shape = weight.shape.clone().unwrap();
+    let channels: [usize; 2] = [shape[1], shape[0]];
+
+    for (key, value) in curr.attrs.iter() {
+        match key.as_str() {
+            "kernel_shape" => kernel_shape = value.clone().into_i64s(),
+            "strides" => strides = value.clone().into_i64s(),
+            "pads" => pads = value.clone().into_i64s(),
```

Comment:
Just realized we don't parse the `auto_pad` attribute (mutually exclusive) but that applies to all conv so no worries we can keep it as the current limitation. Don't think it's used that much.